themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1792,6 +1791,12 @@ html body.tc-body.tc-single-tiddler-window {
 	width: 100%;
 }
 
+.tc-controlpanel svg.tc-image-button{
```

Comment:
typo: `.tc-control-panel` 
and 
`.tc-controlpanel .tc-image-button`  ... imo has the same effect.